modules/data-store/src/data-store.js

```diff
@@ -0,0 +1,147 @@
+
+import serializer from "abitbol-serializable/lib/serializer";
+import minimatch from "minimatch";
+
+
+/**
+ * Stores project's entities and blobs.
+ */
+class DataStore {
+
+    constructor() {
+        this.entitiesByPath = {};
+        this.entitiesByUuid = {};
+    }
+
+    /**
+     * Add an entity to the store.
+     *
+     * @param {SerializableClass} entity The entity
+     * @param {string} path Path where the entity will be stored
+     *                      (default=``"/"``)
+     * @return {undefined}
+     */
+    addEntity(entity, path = "/") {
+        if (!this.entitiesByPath[path]) {
+            this.entitiesByPath[path] = [];
+        }
+        entity.$data._path = path; // eslint-disable-line
+        this.entitiesByPath[path].push(entity);
+        this.entitiesByUuid[entity.id] = entity;
+        // TODO add "path" property to entity and assign the path value
+        // TODO emit an event ("entity-added" with the entity in param)
+    }
+
+
+    /**
+     * Remove an entity to the store.
+     *
+     * @param {SerializableClass|string} entity The entity or its id
+     * @return {undefined}
+     */
+    removeEntity(entity) {
+        // const id = (entity instanceof String) ? entity : entity.id;
+        let id;
+        let realEntity;
+        if ((typeof entity) === "string") {
+            id = entity;
+            realEntity = this.entitiesByUuid[id];
+        } else {
+            id = entity.id; // eslint-disable-line
```

Comment:
We may have to remove the eslint rule that seems too boring... Also prefer disabling only ONE rule on the line instead of all rules (`eslint-disable-line prefer-destructuring`)

Reply:
Done